extract_otp_secret_keys.py

```diff
@@ -85,7 +88,9 @@ def print_qr(data):
     if verbose: print(payload)
 
     # pylint: disable=no-member
+    i = int(0)
     for otp in payload.otp_parameters:
+        i+=1
```

Comment:
Code formatting: I've used spaces around operations in this project.
I suggest
`i += 1`
Here and also in other parts of this pull request